doc/user/content/integrations/redpanda-cloud.md

```diff
@@ -0,0 +1,108 @@
+
+    c. Click on the **Download** button next to the **Kafka API TLS**
+
+    d. Next base64 encode the certificate and save somewhere safe. You'll need it later in this tutorial.
+
+
+    One way of doing this is to use the following command:
+    ```shell
+    cat <redpanda-broker-ca>.crt | base64
+    ```
+
+4. #### Create a topic
+    To start using Materialize with Redpanda, you need to create a Materialize source over a Redpanda topic.
+
+    If you already have a topic created, you can skip this step.
+
+    Otherwise, you can install Redpanda on your client machine from the previous step and create a topic. You can find more information about how to do that [here](https://docs.redpanda.com/docs/reference/rpk-commands/#rpk-topic-create).
+
+
+5. #### Create a source in Materialize
+    a. Open the [Redpanda Cloud console](https://vectorized.cloud/) and select your cluster
+
+    b. Click on **Overview**
+
+    c. Copy the URL under **Cluster hosts**. This will be your `<broker-url>` going forward
+
+    d. From the _psql_ terminal, run the following command. Replace `<redpanda_cloud>` with whatever you want to name your source. The broker URL is what you copied in step c of this subsection. The `<topic-name>` is the name of the topic you created in Step 4. The `<your-username>` and `<your-password>` is from _Store a new secret_ under Step 2.
+    ```sql
+      CREATE SECRET redpanda_username AS '<your-username>';
+      CREATE SECRET redpanda_password AS '<your-password>';
+      CREATE SECRET redpanda_ca_cert AS  decode('<redpanda-broker-ca-cert>', 'base64'); -- The base64 encoded certificate
+      CREATE CONNECTION <redpanda_cloud>
+        FOR KAFKA
+          BROKER '<redpanda-broker-url>',
+          SASL MECHANISMS = 'SCRAM-SHA-256',
+          SASL USERNAME = SECRET redpanda_username,
+          SASL PASSWORD = SECRET redpanda_password,
+          SSL CERTIFICATE AUTHORITY = SECRET redpanda_ca_cert;
+      CREATE SOURCE <topic-name>
+        FROM KAFKA CONNECTION redpanda_cloud TOPIC '<topic-name>'
+        FORMAT BYTES;
+    ```
+
+    e. If the command executes without an error and outputs _CREATE SOURCE_, it means that you have successfully connected Materialize to your Redpanda cluster. You can quickly test your connection by running the following statement:
+    ```sql
+      SELECT convert_from(data, 'utf8') from topic_name;
+    ```
+
+    **Note:** The example above walked through creating a source which is a way of connecting Materialize to an external data source. We used the `WITH` options to provide the username and password and connect to Redpanda using SASL. For input formats, we used `text`, however, Materialize supports various other options as well. For example, you can ingest Kafka messages formatted in [JSON, Avro and Protobuf](/sql/create-source/kafka/#supported-formats). You can find more details about the various different supported formats and possible configurations [here](/sql/create-source/kafka/).
```

Comment:
```suggestion
    **Note:** The example above walked through creating a source, which is a way of connecting Materialize to an external data source. We created a connection to Redpanda using SASL authentication, using credentials securely stored as secrets in Materialize's secret management system. For input formats, we used `bytes`, however, Materialize supports various other options as well. For example, you can ingest Redpanda messages formatted in [JSON, Avro and Protobuf](/sql/create-source/kafka/#supported-formats). You can find more details about the various different supported formats and possible configurations [here](/sql/create-source/kafka/).
```

Reply:
The `WITH` options are no longer part of the syntax, so the blurb also needs to be adjusted in e.g. the MSK guide. Left a suggestion!